lib/resolvers.js

```diff
@@ -0,0 +1,97 @@
+class Resolvers {
+  constructor(context) {
+    this.Kuzzle = context.accessors.sdk;
```

Comment:
Variable starting with a capital letter are usually constructors function.  
Also we are used to avoid storing it in a variable called `kuzzle` to avoid confusion with the internal `kuzzle` object